lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -162,7 +162,7 @@ def instructor_dashboard_2(request, course_id):
             unicode(course_key), len(paid_modes)
         )
 
-    if settings.FEATURES.get('INDIVIDUAL_DUE_DATES') and access['instructor']:
+    if configuration_helpers.get_value('INDIVIDUAL_DUE_DATES', True) and access['instructor']:
```

Comment:
The default site-wide config should be kept.

```suggestion
    if configuration_helpers.get_value(
        'INDIVIDUAL_DUE_DATES',
        settings.FEATURES.get('INDIVIDUAL_DUE_DATES')
    ) and access['instructor']:
```

Reply:
Fixed. Thanks @OmarIthawi